### ISSUE TITLE
Add ability to clone charts/dashboards

### ISSUE THREAD

**Issue by smacker** [src-d/sourced-ui#283]
It was a huge pain to create dashboards that are similar to existing ones during demo.

//cc @rpau

**Comment by carlosms** [comment]
Are you proposing a CLI tool, or adding support in UI?

**Comment by smacker** [comment]
UI. Right now we have buttons "delete" and "export" on dashboard list and only "delete" on charts list. It would be super useful to add "clone" in the lists.

**Comment by ricardobaeta** [comment]
Makes sense to me. With [this icon I believe](https://fontawesome.com/icons/clone?style=regular)

**Comment by smola** [comment]
@smacker would they clone backing queries too?

**Comment by smacker** [comment]
@smola yep. For our use cases, we needed cloning because we were changing the query, not the chart configuration itself.

**Comment by rpau** [comment]
@smola yes.  +1 to @smacker

**Comment by ricardobaeta** [comment]
Just to hence the still valid alignment of this initiative with our researched [usability key finding](https://docs.google.com/document/d/1SGk5harg56Fe5m-mtc1ACvWcv3DvLuP3cRitqCLwCAY/edit#bookmark=id.cjmmlgrgfnie).

![image](https://user-images.githubusercontent.com/1746146/67221467-665e4e80-f423-11e9-80f6-f432c00d69de.png)
